docs/libraries/keypadc.rst

```diff
@@ -13,7 +13,7 @@ It supports multi-key presses, and can be used for extremely responsive input co
 .. contents:: :local:
    :depth: 3
 
-Helpful key dection
+Helpful key detection
```

Comment:
I just noticed, why is the word "Helpful" here? Is it just me thinking that it sounds weird with it?

Reply:
I will change that too.